packages/snippets/test/snippet_parser_test.dart

```diff
@@ -211,11 +247,51 @@ class DocumentedClass {
   /// {@tool snippet}
   /// Description
   /// ```dart
-  /// void memberSample() {
-  /// }
+  /// String elementName = 'DocumentedClass.member';
   /// ```
   /// {@end-tool}
   void member() {}  
 }
 ''');
 }
+
+File _createDartpadSourceFile(Directory tmpDir, FileSystem filesystem, Directory flutterRoot, {bool linked = false}) {
+  final File linkedFile = filesystem.file(path.join(flutterRoot.absolute.path, 'linked_file.dart'))..createSync(recursive: true)..writeAsStringSync('''
```

Comment:
Minor nit: might want to break up this line as it is a little hard to read in one chunk.

Reply:
Sure, done.